main.go

```diff
@@ -36,12 +37,18 @@ func main() {
 			Description: "Gets specific config for service.",
 			Run: func(args []string) {
 				var name string
-				config := parseFlags("get", args, func(fs *flag.FlagSet) {
+				config, parseErr := parseFlags("get", args, false, func(fs *flag.FlagSet) []string {
+					required := []string{"name"}
+
 					fs.StringVar(&name, "name", "", "Name of the config item to retrieve.")
+
+					return required
 				})
 
+				check(config.Logger, parseErr, "", 2)
```

Comment:
Would be nice to use something other than empty string for the message here. At the moment it reports as:

`; mandatory flag 'name' is not set or is empty`

How about e.g. "unable to parse config" or something like that? This would give:

`unable to parse config; mandatory flag 'name' is not set or is empty`